tests/build-job-scope.test.ts

```
@@ -1,6 +1,6 @@
 import { randomUUID } from 'node:crypto'
 import { afterAll, beforeAll, describe, expect, it } from 'vitest'
-import { BASE_URL, USER_ID_2, getSupabaseClient, resetAndSeedAppData, resetAppData } from './test-utils.ts'
+import { BASE_URL, USER_ID_2, createDirectApiKeyWithBindings, getSupabaseClient, resetAndSeedAppData, resetAppData } from './test-utils.ts'
```

ESLint

[error] Expected "createDirectApiKeyWithBindings" to come before "USER_ID_2".

(perfectionist/sort-named-imports)